README.rst

```diff
@@ -175,6 +175,14 @@ For documentation for this project - you're reading it. ;-)
 
 For in-editor documentation aimed at the user, this is in the help.html file.
 
+Metrics
+-------------
+
+In order to help to track how much features of the editor are being used all buttons
```

Comment:
I think track suggest we're enabling something beyond just measuring (track has implications of following, whereas this really just allows you to add measurements).

How about something like 

"in order to help developers measure the way features of the editor are being used, all buttons and actions have an 'action' class added to them. Measurement can the be added by anyone deploying the editor through inclusion of a script that attaches events to these actions and counts them."

As these are docs for the public editor that anyone could use, making statements about the nature of the script and what it does/doesn't send seems inappropriate here: these are more docs for the technology. When we deploy a version that collects anonymous information about the number of events on buttons, like compiles, etc, we need to explain that as part of the deployment/microbit.org stuff.

Reply:
This sounds good to me, I have updated the README with these changes.